ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -707,13 +718,14 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           </FlexRow>
         </WorkspaceEditSection>
         {this.isMode(WorkspaceEditMode.Duplicate) &&
-        <FlexRow>
+        <WorkspaceEditSection header='Options for duplicate workspace'
+          >
           <CheckBox
-                 style={{height: '.66667rem', marginRight: '.31667rem', marginTop: '1.2rem'}}
-          onChange={v => this.setState({cloneUserRole: v})}/>
-          <WorkspaceEditSection header='Copy Original workspace Collaborators'
-            description='Share cloned workspace with same collaborators'/>
-        </FlexRow>
+            style={styles.checkboxStyle}
+            label='Share workspace with the same set of collaborators'
+            labelStyle={styles.text}
```

Comment:
this results in the same style as the original styling? even with no margins set?

Reply:
I do like the appearance of this better than the old appearance, including the larger checkbox.